src/days_in_year.js

```diff
@@ -0,0 +1,23 @@
+const daysInYear = (year) => {
+    try {
```

Comment:
використовуй 2 пробіли замітсь 4
```suggestion
  try {
```